app/Jobs/NotifyUserOfNewGistComments.php

```diff
@@ -68,7 +68,7 @@ private function handleComment($comment, $gist, $user)
 
     private function commentNeedsNotification($comment, $user)
     {
-        if ($comment['updated_at'] < $user['created_at']) {
+        if ($comment['updated_at'] < $user['created_at'] || $comment['user']['id'] !== $user['owner']['id']) {
```

Comment:
Actually, I was way off. I don't know where I got `$user['owner']['id']` from...That's a null value...

I think I had entries added to my dev database and didn't see them pop up again when I re-ran the command so I thought I was good.

Does this make sense now? `==` because they are different types?